README.md

```diff
@@ -37,6 +37,209 @@
 * Can work with Extensions
 * Can work with making a Class and Enums
 
+## JF NEW TEXT
+Just as in any object-oriented programming language, classes in Swift encapsulate data and functionality. In practical terms, that means properties that are variables (`var`) and constants (`let`) as well as methods. (Class methods are functions that are declared within a class. Functions can also be declared outside of classes.)
+
+>For more on classes insert Xref to Classes
+
+## Create a New Class - Person
+You can create a Person class.
+
+Encapsulated in that class will be three properties: name, height, and favoriteFood. Here are the property declarations:
+
+		var name: String
+		var height: Double
+		var favoriteFood: String
+
```

Comment:
I just got home, I'm going to finish reviewing this now. I will let you know when my review is complete, when I do.. I would ask that you make all the necessary changes I'm pointing out here. When you make those changes on your branch they will reflect back here so I can re-review them and then most likely at that point I will merge your branch in.

I would ask that you work in the `readme.md` file.

As well, within your readme.md file.. if you can only leave at the top the Learning Objectives, in fact.. remove where it states the student should be able to, it should only read as Learning Objectives and fill out any bullet points necessary that you think reflect what the objectives are of the lesson you wrote.

The "what the student knows at this point" as well as my outline notes can be completely removed and your content should be put in place.